lib/psi.rb

```diff
@@ -657,7 +657,9 @@ def generateKiva(os_model, walls, floors, edges)
         # Retrieve surface (standard) construction
         s.setAdjacentFoundation(floors[id][:foundation])
         s.setConstruction(s.construction.get)
-        s.createSurfacePropertyExposedFoundationPerimeter(arg, floors[id][:exposed])
+        if floors[id][:exposed] > 0.01
+          s.createSurfacePropertyExposedFoundationPerimeter(arg, floors[id][:exposed])
```

Comment:
Kiva/EnergyPlus don't like exposed perimeters == 0 (e.g. core zone slab with _foundation_ as outside boundary condition). Requires more testing to circumscribe similar critical cases ...